### PR TITLE
Fix typos discovered by codespell

### PR DESCRIPTION
**Pull request by cclauss**
https://pypi.org/project/codespell

% `codespell --ignore-words-list=assertin,clen,windowz --skip="*.patch" --write-changes`

All GitHub Actions pass except for:
* #3077

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ a Python web server.
 It automatically supports dependencies on most pure Python packages. For other
 packages, including those that depend on C code, a special "recipe" must be 
 written to support cross-compiling. python-for-android comes with recipes for
-many of the mosty popular libraries (e.g. numpy and sqlalchemy) built in.
+many of the mostly popular libraries (e.g. numpy and sqlalchemy) built in.
```
Comment (cclauss) [inline README.md]:
```suggestion
many of the most popular libraries (e.g. numpy and sqlalchemy) built in.
```

File: doc/source/index.rst [hunk 2]
```diff
@@ -27,7 +27,7 @@ a Python web server.
 It automatically supports dependencies on most pure Python packages. For other
 packages, including those that depend on C code, a special "recipe" must be
 written to support cross-compiling. python-for-android comes with recipes for
-many of the mosty popular libraries (e.g. numpy and sqlalchemy) built in.
+many of the mostly popular libraries (e.g. numpy and sqlalchemy) built in.
```
Comment (cclauss) [inline doc/source/index.rst]:
```suggestion
many of the most popular libraries (e.g. numpy and sqlalchemy) built in.
```

**Review by cclauss — COMMENTED** [review]
mostly --> most x 2.

**Review by AndreMiras — DISMISSED** [review]
Great catch thanks for looking into it.
Can the code spelling PR be merged into one?
Ping me once it's ready I'd be happy to merge it

**Comment by cclauss** [comment]
CHANGELOG.md added.

**Review by AndreMiras — APPROVED** [review]
LGTM, will merge that one tomorrow so you can sneak more changes if needed

**Review by AndreMiras — APPROVED** [review]

**Pull request merged**